health/REFERENCE.md

```diff
@@ -370,6 +371,22 @@ increasing. Eventually, the comparison will find the averages in the two time-fr
 However, the issue was not resolved, it's just a matter of the newer data "polluting" the old. For such alarms, it's a
 good idea to tell Netdata to not clear the notification, by using the `no-clear-notification` option.
 
+#### Alarm line `label`
+
+Defines the list of labels expected on host, for example, case I want apply an alarm only on Netdata slave, I can specify
+this using the label `_is_master`
+
+```yaml
+label: _is_master = false
+```
+ 
+This option accepts [Netdata simple patterns](../libnetdata/simple_pattern/), so, case you wanna apply an alarm to either
+a master or a slave, you can write:
+
+```yaml
+label: _is_master = *
```

Comment:
how about AND and OR ?

Reply:
several labels, like

(_is_master = false AND _is_slave = false) OR (_is_master = true AND _is_slave = true)

Reply:
It is completely possible, we are using simple pattern, so the users are able to do any valid combination inside label.

Reply:
I  am waiting the final definition about this internal label to change the documentation.

Reply:
Not it is not possible. I mean AND/OR between labels.

Reply:
What is separator btw? How do i separate one label from other?

That is example, again, i want to apply alarm to:
 - master and salve servers
OR
 - standalone servers

i need something like this:

`label: (_is_master = "true" AND _is_slave = "true") OR (_is_master = "false" AND _is_slave = "false")`

Reply:
All right, I set inside my netdata.conf the following labels:

```
[host labels]
        label1 = 123
        label2 =456
        label3= 789
        label4 = 10
```

And I tested with success the following lines:

```
label: label1=123 OR label25=456
label: label1=123 AND label2=456
```

About `_is_slave`, it is not set in any place, we only have `_is_master` merged until now, probably, this is the problem in your side.

Reply:
No, it seems i wasnt clear. I havent tested. I read your documentation. It wasnt clear that OR/AND syntax is working.

So:
 - you can add any number of `label:`? i didnt see it in the docs
 - if i use simple pattern (expression with spaces) why symbols should i use to make it explicit? `"`?

```
label: label1="a* b* !c* *" AND label2="a* b* !c* *"
```

what is logical operator between `label:`? OR?

Reply:
@thiagoftsm so it is possible to use `AND/OR` and `()` to group things inside `label:`?

If so this should be in the docs.

Reply:
I got now, I will extend the documentation given better explanation, thanks!

Reply:
I extended the documentation as you requested @ilyam8 , let me know whether I am clear now on it.

Reply:
> if i use simple pattern (expression with spaces) what symbols should i use to make it explicit? "?

> label: label1="a* b* !c* *" AND label2="a* b* !c* *"

like that?